kanopi/kanopi-components
# Component Library Tooling Update

## Description
> As a package maintainer, I need a consistent set of tools to support developing, maintaining compatability, and limiting regression.

## Related Tickets

* [Component Library Tooling Update](https://kanopi.teamwork.com/app/tasks/29341581)


## Steps to Validate
1. Pull this branch, Docker and XCode are required to run the `makefile` tests 
1. Run `make` and verify that 6 log files are created (and are Git excluded) in the project root - one each for PHPCS and PHPUnit of the supported PHP Versions, 8.0, 8.1, and 8.2
2. Check the log files and ensure they do not show any errors; they will all contain a test/report summary, yet should throw no error codes to the CLI
